### PR TITLE
added work section

### PR DESCRIPTION
**Pull request by oluwajuwon8**
I added a work section to my portfolio.

### CODE REVIEW

**Review by Alexoid1 — CHANGES_REQUESTED** [review]
STATUS: INVALID :x:
## Hello @oluwajuwon8 ✋🏼

- [ ]  Please fix linter errors. After you get all linters to look with a green checkmark (:white_check_mark:) go ahead send for another review with the same pull request.(https://github.com/microverseinc/linters-config).
- [ ] Kindly follow GitHub-flow guidelines. Remember that your branch name should be descriptive so that others can see what is being worked on. Watch this [video](https://www.youtube.com/watch?v=PrIY8sYwe90) to learn more about it.
- [ ] Please add a descriptive pull request with a description of the project.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

Cheers and Happy coding! :computer::clap:
If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request.

File: style.css [hunk 1]
```diff
@@ -5,17 +5,14 @@ header {
...
 
-body {
-  background-color: #16100c;
-}
-
 ul {
   list-style: none;
 }
 
 .content {
```
Comment (AbdelrhmanAmin) [inline style.css:12]: - [ ] The content of the project is aligned with flexbox

**Review by AbdelrhmanAmin — CHANGES_REQUESTED** [review]
Hi @oluwajuwon8, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
None.

Happy hacking! :man_technologist: :woman_technologist: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by AbdelrhmanAmin** [comment]
Sorry for the duplicated comment. 😅

File: style.css [hunk 2]
```diff
@@ -5,9 +5,10 @@ header {
   background-color: rgb(29, 26, 26);
 }
 
-body {
-  background-color: #16100c;
-}
+/* body {
+  display: flex;
+  flex-direction: column;
+} */
```
Comment (TedLivist) [inline style.css:11]: - [ ] Please remove commented code. It makes your code more readable.

File: index.html [hunk 3]
```diff
@@ -1,47 +1,163 @@
...
+        when an unknown printer took a dummy text.
+      </p>
+    </div>
+<div class="">
+    <ul class="grid-container">
+      <li class="grid-item">Css</li>
+      <li class="grid-item">Html</li>
+      <li class="grid-item">bootstrap</li>
+      <li class="grid-item">Ruby</li>
+    </ul></div>
```
Comment (TedLivist) [inline index.html:67]: - [ ] You seem to have introduced a formatting issue. Please correct it. It was not present during the previous review.

**Review by TedLivist — CHANGES_REQUESTED** [review]

File: style.css (hunk 2 above)
Comment (TedLivist) [inline style.css:11]: - [ ] Please remove commented code. It makes your code more readable.

File: index.html (hunk 3 above)
Comment (TedLivist) [inline index.html:67]: - [ ] You seem to have introduced a formatting issue. Please fix the indentation problem. It was not present during the previous review.

**Review by TedLivist — CHANGES_REQUESTED** [review]
Hi @oluwajuwon8,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**